src/contexts/OrderContext/index.js

```diff
@@ -297,28 +297,35 @@ export const OrderProvider = ({ Alert, children, strategy, isAlsea, isDisableToa
       }
       try {
         setState({ ...state, loading: true })
+        const options = {}
         state.loading = true
-        let headers = {
+        options.headers = {
           'X-Socket-Id-X': socket?.getId()
         }
         const countryCode = changes?.country_code && changes?.country_code !== state?.options?.address?.country_code
           ? changes?.country_code
-          : countryCodeFromLocalStorage ?? changes?.country_code ?? state?.options?.address?.country_code
+          : countryCodeFromLocalStorage
```

Comment:
Why do you delete the fallbacks options?

Reply:
solved